shared/src/common/enums/exceptions/exception-message.enum.ts

```diff
@@ -31,5 +31,7 @@ enum ExceptionMessage {
   INTERVIEWEE_CAN_NOT_BE_DELETED = 'This user is forbidden to delete because he has interviews.',
   PROTECTED_GROUP_UPDATE = 'This group is protected and cannot be updated.',
   PROTECTED_GROUP_DELETE = 'This group is protected and cannot be deleted.',
+  MENTOR_CANT_BE_STUDENT = 'A course mentor cannot become a student of the course',
```

Comment:
I would like to know coaches opinion. Should we here change a name of constant to `MENTOR_CANT_BE_STUDENT_FOR_PARTICULAR_COURSE` and same with STUDENT_CANT_BBE_MENTOR_FOR_PARTICULAR_COURSE. It os longer, seems redundant but in my opinion it expresses more sense then a short version

Reply:
`MENTOR_CANT_BE_STUDENT_FOR_SAME_COURSE` would be a little bit shorter 🤔

Reply:
it's also okay